packages/@aws-cdk/cx-api/lib/assets.ts

```diff
@@ -89,10 +93,16 @@ export interface ContainerImageAssetMetadataEntry extends BaseAssetMetadataEntry
    * Note, this is only the repository name, without the registry and
    * the tag parts.
    *
-   * @default automatically derived from the asset's ID.
+   * @default - automatically derived from the asset's ID.
    */
   readonly repositoryName?: string;
 
+  /**
+   * The docker image tag to use for tagging pushed images.
+   * @default "latest"
```

Comment:
`@default Don't care what the tag will be`

Reply:
updated.